src/radical/saga/adaptors/cobalt/cobaltjob.py

```diff
@@ -271,22 +249,21 @@ def _cobaltscript_generator(url, logger, jd, ppn, is_cray=False, queue=None,
   #         % (number_of_processes, number_of_nodes, processes_per_host,
   #             (number_of_nodes * processes_per_host)), rse.NoSuccess, logger)
 
-    # Other funky math checks should go here ~
-
     # Set number of nodes
     cobalt_params += '#COBALT --nodecount %d\n' % number_of_nodes
 
     # Set the total number of processes
     cobalt_params += '#COBALT --proccount %d\n' % number_of_processes
 
-    # The Environments are added at the end because for now
-    # Cobalt isn't supporting spaces in the env variables...
-    # Which mess up the whole script if they are at the begining of the list...
```

Comment:
Is that information outdated?

Reply:
I didn't find either confirming or opposite, thus thought since we keep such order of parameters then it's ok to eliminate this comment. Should I bring it back?

Reply:
I don't understand the original comment really.  How would 
```
#COBALT FOO=BAR BUZ
#COBALT -queue foo
```
be a problem?  I can understand that the `env` might be a problem, but I don't see why the order would be.  
So, I am not sure if that is worthwhile keeping *shrug* I am fine either way.